flink-end-to-end-tests/test-scripts/test_streaming_elasticsearch.sh

```diff
@@ -45,4 +45,5 @@ $FLINK_DIR/bin/flink run -p 1 $TEST_ES_JAR \
   --index index \
   --type type
 
-verify_result 20 index
+# 40 index requests and 20 final update requests
+verify_result 60 index
```

Comment:
so, is the reason why the e2e tests weren't failing because the incompatibility was only detected on update requests?

Reply:
yes, that was the reason